docs/features/nixl_connector_usage.md

```diff
@@ -0,0 +1,154 @@
+# NixlConnector Usage Guide
+
+NixlConnector is a high-performance KV cache transfer connector for vLLM's disaggregated prefilling feature. It provides fully asynchronous send/receive operations using the NIXL library for efficient cross-process KV cache transfer.
+
+## Prerequisites
+
+### Installation
+
+Install the NIXL library:
+
+- Refer to [NIXL official repository](https://github.com/ai-dynamo/nixl) for installation instructions
+- The required NIXL version can be found in [requirements/kv_connectors.txt](../../requirements/kv_connectors.txt) and other relevant config files
+
+### Transport Configuration
+
+NixlConnector uses NIXL library for underlying communication, which supports multiple transport backends. UCX (Unified Communication X) is the primary default transport library used by NIXL. Configure transport environment variables:
+
+```bash
+# Example UCX configuration, adjust according to your enviroment
+export UCX_TLS=all  # or specify specific transports like "rc,ud,sm"
```

Comment:
not sure if overloading with info but you can also exclude transports with
`export UCX_TLS=^cuda_ipc`

Reply:
good:-)